reference/architecture/index.md

```diff
@@ -128,8 +128,11 @@ By default Spinnaker binds ports according to the following table
 | Fiat        | 7003              |
 | Front50     | 8080              |
 | Gate        | 8084              |
+| Gate x509   | 8085              |
```

Comment:
Is this saying that when you use x509 authentication for Gate it instead listens on port 8085?  I'm using this and it's still listening in 8084...when does it listen on 8085?

Reply:
Need to check with SME on this as I haven't enabled x509 and simply based this on internal doc. Stay tuned : )